Shared/Constants.cs

```diff
@@ -0,0 +1,20 @@
+﻿/// <summary>
+/// Useful constants definitions controlled by conditional compilation
+/// </summary>
+
+internal static class Constants
+{
+#if KK
+    internal const string Prefix = "KK";
+    internal const string GameName = "Koikatsu";
+    internal const string StudioProcessName = "CharaStudio";
+    internal const string MainGameProcessName = "Koikatu";
+    internal const string MainGameProcessNameSteam = "Koikatsu Party";
+    internal const string VRProcessName = "KoikatuVR";
+    internal const string VRProcessNameSteam = "Koikatsu Party VR";
```

Comment:
The game name constants are now available from KKAPI.KoikatuAPI so there's no need to keep them locally. What's left in this file should be combined with Info or inlined.

Reply:
Erased the Constants file and used the definitions in KKAPI.KoikatuAPI.